RELEASE.md

```diff
@@ -0,0 +1,24 @@
+# Releasing
+
+Before releasing a new version of packages to the NPM and GitHub, Its advisable to follow the below steps in order
+
+- Update the dependencies `npm run clean`
+- Bootstrap the deps using `npm run bootstrap`
+- Make sure all tests passes `npm run test`
+
+### Releasing all packages
+
+To publish all the packages run `npm run publish`. It will run lerna publish (we use --independent) so it will prompt the version number for
```

Comment:
Do you mean `npm run release` instead of `npm run publish`?

Reply:
oops yeah, you are right.